autowarefoundation/autoware.universe
# feat(planning/control modules): enable logging_level_configure

## Description

Based on https://github.com/autowarefoundation/autoware.universe/pull/5131

This is to provide logging_level_configure service in some nodes. This service enables users to change the logger level in runtime via service.

**How to test**

Run psim and check if the service `<node_name>/config_logger` is running. You can see the logger level will change when you send a service as in the picture of https://github.com/autowarefoundation/autoware.universe/pull/5131

## Related links

None

## Tests performed

run psim.

## Notes for reviewers



## Interface changes



## Effects on system behavior



## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.


## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.


## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.


After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/
